### PR TITLE
Adding Multi-GPU Training Script to Existing SSL Tutorial

### PR DESCRIPTION
**Pull request by finalelement**
Fixes # .

### Description
The self-supervised learning tutorial will be able to support multi-gpu training with the new script from this PR.

### Checks

- [x] Avoid including large-size files in the PR.
- [x] Clean up long text outputs from code cells in the notebook.
- [x] For security purposes, please check the contents and remove any sensitive info such as user names and private key.
- [ ] Ensure (1) hyperlinks and markdown anchors are working (2) use relative paths for tutorial repo files (3) put figure and graphs in the `./figure` folder
- [ ] Notebook runs automatically `./runner.sh -t <path to .ipynb file>`

### CODE REVIEW

File: self_supervised_pretraining/README.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ are based on augmentations of different types. The second is notebook 'ssl_finet
 the pre-trained weights generated from the first script and performs fine-tuning on a fully supervised
 task.
 
+The tutorial also provides a multi-GPU version of the SSL training, the code to run multi-gpu SSL can
```
Comment (mingxin-zheng) [inline self_supervised_pretraining/README.md:9]: This line has two sentences.

File: self_supervised_pretraining/README.md [hunk 2]
```diff
@@ -132,3 +135,24 @@ L1.
...
+
+It can be configured to launch on more GPU's by adding the relevant `CUDA Device` ID in `CUDA_VISIBLE_DEVICES`
+and increasing the total count of GPU's `--nproc_per_node`
+
+Below are expected run time comparison between a single GPU and multi-GPU benchmarked on A100 GPU's
+
+
+| Hardware   | Training Time  |
+|------------|----------------|
+| 1 GPU A100 | Under Progress |
```
Comment (mingxin-zheng) [inline self_supervised_pretraining/README.md:157]: Will the training time for single A100 be updated in the PR

File: self_supervised_pretraining/multi_gpu/mgpu_ssl_train.py [hunk 3]
```diff
@@ -0,0 +1,364 @@
...
+
+    # Set Paths for running SSL training
+    parser.add_argument("--data_root", default="/workspace/datasets/tcia/", type=str, help="path to data root")
+    parser.add_argument(
+        "--json_path",
+        default="./datalists/tcia/dataset_split.json",
+        type=str,
+        help="Json file path for list of data samples",
+    )
+    parser.add_argument("--logdir_path", default="/to/be/defined", type=str, help="output log directory")
```
Comment (mingxin-zheng) [inline self_supervised_pretraining/multi_gpu/mgpu_ssl_train.py:62]: maybe using "." as the default?

**Review by mingxin-zheng — APPROVED** [review]

File: self_supervised_pretraining/README.md (hunk 2 above)
Comment (finalelement) [reply self_supervised_pretraining/README.md:157]: Yes

File: self_supervised_pretraining/README.md (hunk 1 above)
Comment (finalelement) [reply self_supervised_pretraining/README.md:9]: Thank you, good catch, will fix.

File: self_supervised_pretraining/multi_gpu/mgpu_ssl_train.py (hunk 3 above)
Comment (mingxin-zheng) [reply self_supervised_pretraining/multi_gpu/mgpu_ssl_train.py:62]: Feel free ignore this comment.

File: self_supervised_pretraining/README.md (hunk 1 above)
Comment (finalelement) [reply self_supervised_pretraining/README.md:9]: fixed.

File: self_supervised_pretraining/README.md (hunk 2 above)
Comment (finalelement) [reply self_supervised_pretraining/README.md:157]: Cannot be addressed, due to resource issues as of now.

File: self_supervised_pretraining/README.md [hunk 4]
```diff
@@ -132,3 +135,38 @@ L1.
...
+
+To begin training with 2 GPU's please see the below example command for execution of the SSL multi-gpu training
+script:
+
+`CUDA_VISIBLE_DEVICES=0,1 python -m torch.distributed.launch --nproc_per_node=2 mgpu_ssl_train.py --batch_size=8 --epochs=500 --base_lr=2e-4 --logdir_path=/to/be/defined --output=/to/be/defined --data_root=/to/be/defined --json_path=/to/be/defined`
+
+It can be configured to launch on more GPU's by adding the relevant `CUDA Device` ID in `CUDA_VISIBLE_DEVICES`
+and increasing the total count of GPU's `--nproc_per_node`
+
+Below are expected run time comparison between a single GPU and multi-GPU benchmarked on A100 GPU's
```
Comment (mingxin-zheng) [inline self_supervised_pretraining/README.md:152]: Please delete a single GPU expression.

**Pull request merged**